changelog.txt

```diff
@@ -2,6 +2,424 @@
 
 This is an archive of older changelog entries. Most recent entries are maintained in readme.txt
 
+= 3.0.7 =
+
+Release Date: December 23rd, 2015
+
+* Enhancements:
+	* Removes email notifications for OnPage.org indexability check. It caused more issues than it solved.
```

Comment:
The lines you commented on are for old versions. @omarreiss Do we want to clean those up, or might it be better to leave them as is for historical consistency